services/simcore/docker-compose.yml

```diff
@@ -268,6 +323,13 @@ services:
       placement:
         constraints:
           - node.labels.simcore==true
+      resources:
+        reservations:
+          cpus: "0.1"
+          memory: "256M"
+        limits:
+          cpus: "0.2"
+          memory: "1G"
```

Comment:
so why is storage on 0.2 CPU and the director-v0 on 1? 
storage is a lone system, pretty much at the center, while the dv-0 is scaled to 10 replicas (be it for a good reason or not). why that choice?

Reply:
As discussed in whereby